httplinecount/httplinecount.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/gorilla/mux"
+	"net/http"
+	"os"
+
+	"encoding/json"
+	"strconv"
+"errors"
+)
+
+type DataStr struct {
+	Title string `json:"title"`
+	Lines string `json:"lines"`
+}
+
+func CountLines(folder string) (string, error) {
+	f, err := os.Open(folder)
+	if err != nil {
+		return "", errors.New("Can't open the file")
```

Comment:
real reason of the error is missing here:
`errors.New("Can't open the file: " + err.Error()) `